content/en/docs/setup/minikube.md

```diff
@@ -47,9 +47,9 @@ This brief demo guides you on how to start, use, and delete Minikube locally. Fo
     Creating machine...
     Starting local Kubernetes cluster...
     ```
-    For more information on starting your cluster on a specific Kubernetes version, VM, or container runtime, see [Starting a Cluster](docs/setup/minikube/#starting-a-cluster).
+    For more information on starting your cluster on a specific Kubernetes version, VM, or container runtime, see [Starting a Cluster](/docs/setup/minikube/#starting-a-cluster).
```

Comment:
please replace 
/docs/setup/minikube/#starting-a-cluster 
to
minikube.md#starting-a-cluster